### ISSUE TITLE
Wind speeds should be reported in km/h, not m/s as standard

### ISSUE THREAD

**Issue by scstraus** [briis/smartweather#63]
Hi, I tried the wind speed adjustment setting, but no matter what I tried, I seemed to get m/s out of the integration. The problem with this is that the [default as standard appears to be km/h](https://developers.home-assistant.io/docs/core/entity/weather/) so my [weather card was converting it to m/s](https://community.home-assistant.io/t/lovelace-weather-card-with-chart/88816/175?u=scstraus), which made the wind about 30% lower than it really was (as shown on the smartweather site). I think you might need to add conversion to km/h on your side?

**Comment by briis** [comment]
If you use Unit System *metric* in HA, and then set the Wind Unit to km/h, you will get wind values in km/h. I made this on purpose, as I personally wanted the units in m/s (Standard for Denmark) but setting the above option should give you the values as you want.
You can change this from the Integrations page, by clicking options on the SmartWeather integration, and then selecting km/h as Wind Unit.

**Comment by scstraus** [comment]
I tried that but it didn’t appear to change anything.

**Comment by briis** [comment]
Just to check Sean. Please find the Weather Entity in the Developer Tools/States, and check the Wind Values there. Then try and change the unit in the Integration, and see if this also changes the value in the Developer Tools/States.
This is just to check if it is the data coming from the Integration or the Lovelace Card.

What Card are you using to display the data?

**Comment by scstraus** [comment]
I stand corrected. I went and checked it again, and it did indeed change. I think the reason the values were so low was that I was using the wind sensor rather than the wind gust sensor which reports very low values, and wind gust used to be equally low, but lately wind gust has been showing higher numbers so I switched to that as it more accurately seemed to reflect what was going on outside, and at the same time I switched to m/s, so it seems I had conflated the two.. I actually prefer having the sensor in m/s too. The card I'm using is [this one](https://github.com/Yevgenium/lovelace-weather-card-chart).. I've been forking it lately, I think at some point I will just have to put the code in to do the proper coversions in the card.. For now I've just commented it out.
